### DISCUSSION TITLE
How to duplicate a meeting?

### DISCUSSION THREAD

**Discussion by Spsrky** [code4recovery/12-step-meeting-list#728] [Q&A] [answered]
Is it possible to duplicate a meeting as a way of adding a new entry? For example, we have a meeting on on Monday, then they want to extend it to several more days - is there a way to duplicated the existing meeting so I only need to change the day for the new entry? I searched  but couldn't find this. Thanks!

**Comment by Grateful-Viv** [comment, marked as answer]
Not sure if it works, but perhaps you could try a plugin (assuming you’re using Wordpress) like Post Duplicator. See https://wordpress.org/plugins/post-duplicator/

**Comment by Spsrky** [comment]
Hi Viv,

That works fine, thanks! I'll add it in the discussion.

In fellowship and service,

Theron

**Comment by Spsrky** [comment]
Using the WordPress plugin "Post Duplicator" adds another option in the dashboard list of all meetings, so Edit | Trash | View becomes Edit | Trash | View | Duplicate Meeting. Clicking the duplicate option for "Meeting XYZ" then creates a new entry for "Meeting XYZ Copy".
